ega-charts/localega/templates/doa-certs.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if and .Values.config.tls_certs .Values.doa.deploy }}
+{{- $cafile := .Files.Glob (printf "config/certs/%s" .Values.config.tls_ca_root_file)}}
+{{- $javaca := .Files.Glob (printf "config/certs/cacerts") }}
+{{- $doa := .Files.Glob "config/certs/doa*" }}
```

Comment:
This is a java app so we should only need the pkcs12 file i.e. doa.p12

Reply:
Actually, I was going to add support for `pkcs12` in this service as well.

Reply:
Ok. I will leave this one as is then.

Reply:
This is still relevant since the current container will not start in our environment due to UID restrictions.